server/test/lib/device/device.get.test.js

```diff
@@ -94,6 +94,21 @@ describe('Device.get', () => {
       .to.be.instanceOf(Array)
       .and.have.lengthOf(0);
   });
+  it('should get devices filtered by device_feature_selectors', async () => {
+    const stateManager = new StateManager(event);
+    const service = {
+      getLocalServiceByName: fake.resolves({
+        id: 'a810b8db-6d04-4697-bed3-c4b72c996279',
+      }),
+    };
+    const device = new Device(event, {}, stateManager, service, {}, {}, job);
+    const devices = await device.get({
+      device_feature_selectors: 'feature-1,feature-2',
+    });
+    expect(devices)
+      .to.be.instanceOf(Array)
+      .and.have.lengthOf(0);
```

Comment:
Could you test with a real use case to verify that only 2 features are returned?

Reply:
Added